openwisp_monitoring/monitoring/base/models.py

```diff
@@ -671,6 +671,8 @@ def get_query(
         params = self._get_query_params(time, start_date, end_date)
         params.update(additional_params)
         params.update({'start_date': start_date, 'end_date': end_date})
+        if not params.get('organization_id') and self.config_dict.get('filter__all__'):
```

Comment:
maybe we can use just `__all__` for simplicity and clarity to anyone familiar with python (I think we use similar constructs elsewhere)?